src/main/java/jenkins/plugins/git/GitSCMSource.java

```diff
@@ -188,43 +344,55 @@ public String getRemote() {
         return remote;
     }
 
-    @Override
-    public String getRemoteName() {
```

Comment:
hard to see in this diff if all old removed signatures are removed or just moved.